Formula/shtools.rb

```diff
@@ -1,11 +1,11 @@
 class Shtools < Formula
   desc "Spherical Harmonic Tools"
   homepage "https://shtools.github.io/SHTOOLS/"
-  url "https://github.com/SHTOOLS/SHTOOLS/archive/v4.7.1.tar.gz"
-  sha256 "6ed2130eed7b741df3b19052b29b3324601403581c7b9afb015e0370e299a2bd"
+  url "https://github.com/SHTOOLS/SHTOOLS/releases/download/v4.8/SHTOOLS-4.8.tar.gz"
+  sha256 "c36fc86810017e544abbfb12f8ddf6f101a1ac8b89856a76d7d9801ffc8dac44"
   license "BSD-3-Clause"
-  revision 1
-  head "https://github.com/SHTOOLS/homebrew-shtools.git"
+  revision 0
```

Comment:
Why?

Reply:
Sorry, but which part are you referring to? Revision was set to 0 because this is the first release of version 4.8 (and the CI tests failed with "1"). As for head: It is my understanding that this should point to the repo of the project: the previous version mistakenly pointed to the homebrew project with the .rb file in it.

Reply:
0 is the default, you can just leave out that line then.

Reply:
ok. I'll remove that line.